aptos-move/framework/aptos-framework/sources/vesting.spec.move

```diff
@@ -70,19 +81,48 @@ spec aptos_framework::vesting {
         aborts_if total_active_stake < staking_contract.principal;
         aborts_if accumulated_rewards * staking_contract.commission_percentage > MAX_U64;
         aborts_if (vesting_contract.remaining_grant + commission_amount) > total_active_stake;
+        aborts_if total_active_stake < vesting_contract.remaining_grant;
     }
 
     spec accumulated_rewards(vesting_contract_address: address, shareholder_or_beneficiary: address): u64 {
-        // TODO: Uses `total_accumulated_rewards` which is not verified.
-        pragma verify = false;
+        // TODO: A severe timeout can not be resolved.
+        pragma verify_duration_estimate = 1000;
+
+        // This schema lead to timeout
```

Comment:
Do we know why? Is it proved with a large time-limit?

Reply:
This two item both contribute to the timeout, the first one deal much more
aborts_if (vesting_contract.remaining_grant + commission_amount) > total_active_stake;
aborts_if total_active_stake < vesting_contract.remaining_grant;

Reply:
Was it proved when you gave a large time-limit?